.github/workflows/build-containers.yml

```diff
@@ -101,5 +113,5 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
           tags: |
-            cr.seqera.io/spack/${{ env.container }}
-            cr.seqera.io/spack/${{ env.versioned }}
+            195996028523.dkr.ecr.eu-west-1.amazonaws.com/wave/spack/${{ env.container }}
+            195996028523.dkr.ecr.eu-west-1.amazonaws.com/wave/spack/${{ env.versioned }}
```

Comment:
Maybe instead of setting Harbor as a proxy, can we push here both to Harbor and ECR?. Easy :smile:

Reply:
Once the proxy repo in Harbor is set up, you'll be able to pull from Harbor without needing any permissions to access ECR.